client/aci/list.go

```diff
@@ -69,3 +68,69 @@ func (c *Client) ListContainerGroups(ctx context.Context, resourceGroup string)
 
 	return &list, nil
 }
+
+func (c *Client) ListAKSClusters(ctx context.Context, resourceGroup string) (*AKSClusterListResult, error) {
+
+	// make aksApiVersion a new constant
```

Comment:
The comment has been resolved already. please remove it.

Reply:
removed comment